.circleci/config.yml

```diff
@@ -3,14 +3,13 @@ version: 2.1
 executors:
   sgrep-build:
     docker:
-      - image: returntocorp/sgrep-build:2.5
+      - image: returntocorp/sgrep-build:latest
```

Comment:
this should be a tag so Circle can cache it but I want to replace this with a GH action next week as part of the mac build so ok for now